src/image-paste.ts

```diff
@@ -151,6 +151,7 @@ export namespace Import {
       const currentDateString = new Date()
         .toISOString()
         .replace(':', '-')
+        .replace(':', '-')
```

Comment:
It's a shame that `replaceAll` is only available on Node 15 and greater: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll

As far as I know, the latest version of VS code is still based on Node 14.

I think it would be better to use a regular expression: 

```js
new Date().toISOString().replace(/[:.]/g, '-')
```

Reply:
yes, a regexp would be better